docs/home_content.md

```diff
@@ -1,3 +1,14 @@
+## Introduction
+
+Orange is providing a full Design System to build Orange Mobile Application. The objective of the [Orange Design System](https://system.design.orange.com/0c1af118d/p/95b685-ios/) (ODS) is to propose a set of guidelines on how to apply th Orange Brand on mobile applications. The Orange design System also provides a series of components and modules that show in details how to use this in the Orange apps.
+
+The Orange Design System has been implemented in a code library that provides:
+- a Jetpack Compose code library
+- a demo app that can be launched to show the components and modules
+- this demo app also a shows how to use the lib or style existing components
```

Comment:
Another typo : `this demo app also a shows` should be `this demo app also shows`.